src/services/dex.js

```diff
@@ -1069,7 +1104,14 @@ export default {
         this.withdrawNEP5(assetId, quantity)
           .then((res) => {
             if (res.success) {
-              resolve(res.tx);
+              alerts.success('Withdraw Relayed.');
+              neo.monitorTransactionConfirmation(res.tx, true)
+                .then(() => {
+                  resolve(res.tx);
```

Comment:
I would suggest flattening this 'then' as well.

Reply:
ditto